docs/source/contributing/testing-vitest.md

```diff
@@ -0,0 +1,96 @@
+---
+myst:
+  html_meta:
+    "description": "We use Vitest for unit testing in Volto. The popular @testing-library/react is also available for writing your tests. For every feature or component, a unit test is mandatory in Volto core."
+    "property=og:description": "We use Vitest for unit testing in Volto. The popular @testing-library/react is also available for writing your tests. For every feature or component, a unit test is mandatory in Volto core."
+    "property=og:title": "Testing Volto with Vitest"
+    "keywords": "Volto, Plone, frontend, React, testing, Vitest"
+---
+
+# Testing with Vitest
+
+We use `Vitest` for unit testing in Volto.
```

Comment:
It's a brand name, not an inline literal. However, it's a good idea to link to its docs the first time it's mentioned.

```suggestion
We use [Vitest](https://vitest.dev/guide/) for unit testing in Volto.
```